web-api/terraform/template/elasticsearch.tf

```diff
@@ -49,3 +49,27 @@ resource "aws_elasticsearch_domain" "efcms-search" {
     log_type                 = "ES_APPLICATION_LOGS"
   }
 }
+
+resource "aws_elasticsearch_domain" "efcms-app-logs" {
+  domain_name           = "efcms-app-logs-${var.environment}"
+  elasticsearch_version = "7.4"
+
+  cluster_config {
+    instance_type = "t2.small.elasticsearch"
+    instance_count = var.es_instance_count == "" ? "1" : var.es_instance_count
```

Comment:
I’m not sure Terraform allows variables to have blank values — does it? 🤔

Reply:
Found this Terraform doc about [conditional expressions](https://www.terraform.io/docs/configuration/expressions.html#conditional-expressions)

Reply:
you could probably just say `instance_count = var.es_instance_count` here and in the `variables.tf`, declare it as:

```tf
variable "es_instance_count" {
  type = string
  default = null
}
```